frame_test.go

```diff
@@ -127,3 +131,315 @@ func TestFrameReadTooLong(t *testing.T) {
 		t.Fatalf("expected to get header %v got %v", opReady, head.op)
 	}
 }
+
+func Test_framer_writeExecuteFrame(t *testing.T) {
```

Comment:
We need an integration test to validate `<result_metadata_id>` logic. Details of why it was included can be found in ticket [CASSANDRA-10786](https://issues.apache.org/jira/browse/CASSANDRA-10786). I am trying to create a test like it.

Reply:
https://issues.apache.org/jira/browse/CASSANDRA-20028